app/views/course/leaderboards/groups.html.slim

```diff
@@ -0,0 +1,13 @@
+= page_header format_inline_text(@leaderboard_settings.group_leaderboard_title || t('.header'))
+
+= render partial: 'tabs'
+
+- course_groups_points = @course_groups.ordered_by_experience_points.take(display_user_count)
+= render layout: 'table', locals: { header: t('.experience_points_header'),
```

Comment:
Both are fine - was thinking of having `_table` partial will be useful when it comes to styling, but I suppose it might be a little too troublesome. Will revert back to the old structure.